tests/ci/run_ssl_runner_valgrind_tests.sh

```diff
@@ -1,7 +1,9 @@
-#!/bin/bash -ex
+#!/usr/bin/env bash
 # Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 # SPDX-License-Identifier: Apache-2.0 OR ISC
 
+set -ex
```

Comment:
you like `set -exo pipefail` or `set -ex` the most? :)

`tests/ci/run_ssl_asan_tests.sh` use the latter, here the former.

Reply:
This is just keeping the same `bash` flags as were already on the script.  B/c this is switching to invoking bash via `#!/usr/bin/env bash` (instead of `#!/bin/bash`) these flags needed to be moved into a `set` statement.

Reply:
I didn't add pipefail to the scripts that didn't have it previously but I'll just go back and fix them up to all be the same.